build-scripts/spdx-header-chech.sh

```diff
@@ -12,13 +12,14 @@ APPROVED_LICENSES=""
 APPROVED_LICENSES="${APPROVED_LICENSES} GPL-2.0-or-later"
 # Used in src/libbluechi/common/list.h
 APPROVED_LICENSES="${APPROVED_LICENSES} LGPL-2.1-or-later"
+# Used in generated files in pyhirte
+APPROVED_LICENSES="${APPROVED_LICENSES} LGPL-3.0-or-later"
 
 result=0
 found_files="
     $(find -type f -regex ${CHECKED_FILES} \
         -not -path './builddir/*' \
-        -not -path './subprojects/*' \
-        -not -path './doc/arch/*')
```

Comment:
Do the docs really need the SPDX header?

Reply:
Originally there were some source code files under `doc/arch`, which we wanted to omit from header check. But this directory doesn't exist anymore, so I removed it